Plugins/org.commontk.eventbus/ctkNetworkConnectorQXMLRPC.cpp

```diff
@@ -112,7 +112,7 @@ void ctkNetworkConnectorQXMLRPC::stopServer() {
 
 void ctkNetworkConnectorQXMLRPC::registerServerMethod(mafRegisterMethodsMap registerMethodsList) {
     if(m_Server->isListening()) {
-        qDebug("%s", tr("Server is already listening on port %1").arg(m_Server->property("port").toUInt()).toUtf8().data());
+        qDebug("%s", QString("Server is already listening on port %1").arg(m_Server->property("port").toUInt()).toUtf8().data());
```

Comment:
Are error message plugins considered too low level and outside of the translation scope ?

Reply:
Log messages should never be translated, only strings that are displayed to users.

Translating log messages would make it very hard for developers to analyze logs. Developers may not even have access to the translation files that the application used.